.gitignore

```diff
@@ -11,6 +11,7 @@ coverage/
 /data/early-access
 dist
 .next
+package-lock.json
```

Comment:
We'll need to revert this change, as we do need package-lock for our actions processes and for security reasons.

Reply:
👍🏼 If you can also run `git checkout main package-lock.json`

Reply:
Ran the command as well!!

Reply:
Thanks!